.circleci/config.yml

```diff
@@ -1559,6 +1559,7 @@ workflows:
             op-service
             op-supervisor
             op-deployer
+            op-fetcher
```

Comment:
If the contracts exist in `packages/contracts-bedrock` we need another CI job to ensure that the artifacts embedded in the binary match the source.

Reply:
Yes I think we need the ci job regardless of where the contracts exist (since we're embedding the compiled artifacts inside the go package). Will add one

Reply:
Done in https://github.com/ethereum-optimism/optimism/pull/14930/commits/1f44812b5747e13ea1ab9c0646636bacd09239ff